src/reducers/tasks.reducer.js

```diff
@@ -84,10 +83,7 @@ export default (state: State = initialState, action: Action) => {
         Object.keys(project.scripts).forEach(name => {
           const command = project.scripts[name];
 
-          const uniqueTaskId = buildUniqueTaskId(projectId, name);
-
-          draftState[uniqueTaskId] = buildNewTask(
-            uniqueTaskId,
+          draftState[projectId][name] = buildNewTask(
```

Comment:
Does this work? I don't see where `draftState[projectId]` is being created.

Hm I wonder if this is something that `immer` takes care of automatically... even if it does, I think it'd be less surprising if we created the object first.

```js
return produce(state, draftState => {
  const projectTasks = {};

  Object.keys(project.scripts).forEach(name => {
    const command = project.scripts[name];

    projectTasks[name] = buildNewTask(...)

    ...
  })

  draftState[projectId] = projectTasks;
}
```